cavy.js

```diff
@@ -29,8 +29,9 @@ function test(cmd) {
   const commandName = cmd.name();
   const entryFile = cmd.file;
   const skipbuild = cmd.skipbuild;
+  const xml = cmd.xml;
```

Comment:
Philosophical point; I feel like this variable name isn't super friendly, especially once it ends up a couple of functions deep. What about `outputAsXml` or something?

Reply:
agreed! will change.